README.md

```diff
@@ -9,12 +9,14 @@ like `bankai`.
 ## Usage
 ```js
 var choo = require('choo')
+var sw = require('choo-service-worker')
+var clear = require('choo-service-worker/clear')
 
 var app = choo()
 if (process.env.NODE_ENV !== 'production') {
-  app.use(require('choo-service-worker/clear')())
+  app.use(clear())
 }
-app.use(require('choo-service-worker')())
+app.use(sw())
```

Comment:
You definitely want to have a conditional require in this case; else you're shipping extra bytes for no reason (: